src/main/java/frc/robot/commands/defaultCommands/IntakeDefaultCommand.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.commands.defaultCommands;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class IntakeDefaultCommand extends Command {
```

Comment:
You need to implement `end`, and stop the motor. Remember, default commands get interrupted when another command runs which requires the subsystem.